src/main/java/core/basesyntax/AreaCalculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculate {
+    String drawArea();
```

Comment:
it should calculate not draw. overall, drawing is not returning string, drawing is doing printing in console for example
```suggestion
    double calculateArea();
```